test-network-k8s/docs/APPLICATIONS.md

```diff
@@ -0,0 +1,41 @@
+# Working with Applications
+
+## TL/DR: 
+
+```shell
+$ ./network rest-easy 
+Launching fabric-rest-sample application:
+✅ - Ensuring fabric-rest-sample image ...
+✅ - Constructing fabric-rest-sample connection profiles ...
+✅ - Starting fabric-rest-sample ...
+
+The fabric-rest-sample has started. See https://github.com/hyperledgendary/fabric-rest-sample/tree/main/asset-transfer-basic/rest-api-typescript#rest-api for additional usage.
+To access the endpoint:
+
+export SAMPLE_APIKEY=97834158-3224-4CE7-95F9-A148C886653E
+curl -s --header "X-Api-Key: ${SAMPLE_APIKEY}" http://localhost/api/assets
+
+🏁 - Fabric REST sample is ready.
+```
+
+```shell
+$ export SAMPLE_APIKEY=97834158-3224-4CE7-95F9-A148C886653E
+
+$ ./network chaincode invoke '{"Args":["CreateAsset","1","blue","35","tom","1000"]}' 
+
+$ curl -s --header "X-Api-Key: ${SAMPLE_APIKEY}" http://localhost/api/assets | jq 
+[
+  {
+    "Key": "1",
+    "Record": {
+      "ID": "1",
+      "color": "blue",
+      "size": 35,
+      "owner": "tom",
+      "appraisedValue": 1000
+    }
+  }
+]
+
+$ open https://github.com/hyperledgendary/fabric-rest-sample/tree/main/asset-transfer-basic/rest-api-typescript#rest-api 
```

Comment:
should add some more re the ConfigMaps for apps etc... but can add that after as a defect..

Reply:
I ran out of steam a bit setting up a guide for the application development.  It almost seems like this section can wait, simmering for a while as we work out the kinks (if any) with the gateway client development practices.

Should we include to sections here, one for the node client enabled applications and a second for the gateway-client apps?  Or just one?   There are so many "fiddly" parts around the edges of constructing the connection profile documents.

Reply:
exactly;  post-merge activity